fund_store/config/fund_loader_config/FAB/dpif_r3.py

```diff
@@ -0,0 +1,130 @@
+        {
+            "section_name": {"en": "4.3 Dataset information", "cy": ""},
+            "tree_path": "1047.1.4.3",
+            "form_name_json": {"en": "dpif-dataset-information", "cy": ""},
+        },
+        {"section_name": {"en": "5. Future work", "cy": ""}, "tree_path": "1047.1.5", "requires_feedback": None},
+        {
+            "section_name": {"en": "5.1 Future work", "cy": ""},
+            "tree_path": "1047.1.5.1",
+            "form_name_json": {"en": "dpif-future-work", "cy": ""},
+        },
+        {"section_name": {"en": "6. Declarations", "cy": ""}, "tree_path": "1047.1.6", "requires_feedback": None},
+        {
+            "section_name": {"en": "6.1 Declarations", "cy": ""},
+            "tree_path": "1047.1.6.1",
+            "form_name_json": {"en": "dpif-declarations", "cy": ""},
+        },
+    ],
+    "fund_config": {
+        "id": "f493d512-5eb4-11ee-8c99-0242ac120002",
+        "short_name": "DPIF",
+        "welsh_available": False,
+        "owner_organisation_name": "None",
+        "owner_organisation_shortname": "None",
+        "owner_organisation_logo_uri": "None",
+        "ggis_scheme_reference_number": "",
+        "funding_type": "UNCOMPETED",
+        "name_json": {"en": "Digital Planning Improvement Fund", "cy": ""},
+        "title_json": {"en": "Apply for funding to begin your digital planning improvement journey", "cy": ""},
+        "description_json": {
+            "en": "The Digital Planning Improvement Fund will provide local planning authorities with the funding and support to adopt modern planning practices within planning data, digital capabilities and development management software.",
+            "cy": "",
+        },
+    },
+    "round_config": {
+        "id": "ac835965-e8c9-4356-b486-c0c016dbb634",
+        "fund_id": "f493d512-5eb4-11ee-8c99-0242ac120002",
+        "short_name": "R3",
+        "opens": "2024-12-11T09:00:00",
+        "assessment_start": "2024-12-11T09:00:00",
+        "deadline": "2025-01-31T23:59:00",
+        "application_reminder_sent": False,
+        "reminder_date": "2025-01-24T09:00:00",
+        "assessment_deadline": "2025-05-16T23:59:00",
+        "prospectus": "https://www.localdigital.gov.uk/digital-planning/digital-planning-improvement-fund-2024/",
+        "privacy_notice": "https://www.gov.uk/guidance/digital-planning-improvement-fund-privacy-notice",
+        "reference_contact_page_over_email": True,
+        "contact_email": "digitalplanningteam@communities.gov.uk",
+        "contact_phone": "",
+        "contact_textphone": "",
+        "support_times": "9am to 5pm",
+        "support_days": "Monday to Friday (except public holidays)",
+        "instructions_json": {
+            "en": "You must have received an invitation to apply, which should include the fund's prospectus. If you have not received the prospectus, contact digitalplanningteam@communities.gov.uk",
+            "cy": None,
+        },
+        "feedback_link": "",
```

Comment:
Please use this feedback link:

https://forms.office.com/e/USFFFFyaNV

As per [this Slack thread](https://communities-govuk.slack.com/archives/C07S3KW31EE/p1733747344621869)

Reply:
Sorted! Thanks!